rmgpy/data/kinetics/groups.py

```diff
@@ -218,15 +220,42 @@ class together, returning their product as a new kinetics object of
             )
```

Comment:
I agree, it certainly doesn't belong where it's at, but I think we should wait on non-essential cleanup until we decide whether it should be brought back into RMG or not.